tests/runtest.sh

```diff
@@ -397,20 +404,23 @@ function precompile_overlay_assemblies {
                     echo Unable to generate dasm for $filename
                 fi
             else
-                # Precompile any assembly except mscorlib since we already have its NI image available.
-                if [[ "$filename" != *"mscorlib.dll"* ]]; then
-                    if [[ "$filename" != *"mscorlib.ni.dll"* ]]; then
```

Comment:
Removed these lines because they are a special case of a skip list.  Added to skip lists instead.